src/tests/tests.sln

```diff
@@ -0,0 +1,20 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{888888A0-9F3D-457C-B088-3A5042F75D52}") = "tests", "tests.pyproj", "{250C535C-C060-4F0C-BD80-41F2BF373565}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{250C535C-C060-4F0C-BD80-41F2BF373565}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{250C535C-C060-4F0C-BD80-41F2BF373565}.Release|Any CPU.ActiveCfg = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+EndGlobal
```

Comment:
@denfromufa just tried combining to 1 solution and running the solution on a machine without `ptvs`. It loaded the solution no problem, it asked me if i wanted to installed `ptvs` to load up the project. I declined, and it just grays out the project and everything else worked fine. It didn't ask me again if i reloaded the solution. 

The project file needs to be cleaned up though to sanitize the configurations, otherwise it pollutes the current solution. I'm ok with deleting the solution and leaving the `pyproj` without adding it to the current solution on this `pr`.